ui/lib/components/TreeDiagram/NodeDetail/DefaultNodeDetail.tsx

```diff
@@ -1,140 +1,190 @@
+      tikv_huge_table_scan: 'The TiKV read a lot of data. Consider using TiFlash to get better performance if it\'s necessary to read so much data.',
+    },
+  },
+  zh: {
+    tooltip: {
+      duration: '父算子的耗时包含所有子算子的耗时。',
+    },
+    diagnosis: {
+      high_est_error: 'The estimation error is high. Consider checking the health state of the statistics.',
+      disk_spill: 'Disk spill is triggered for this operator because the memory quota is exceeded. The execution might be slow. Consider increasing the memory quota if there\'s enough memory.',
+      pseudo_est: 'This operator used pseudo statistics and the estimation might be inaccurate. It might be caused by unavailable or outdated statistics. Consider collecting statistics or setting variable tidb_enable_pseudo_for_outdated_stats to OFF.',
+      good_filter_on_table_fullscan: 'This Selection filters a high proportion of data. Using an index on this column might achieve better performance. Consider adding an index on this column if there is not one.',
+      bad_index_for_index_lookup: 'This IndexLookup read a lot of data from the index side. It might be slow and cause heavy pressure on TiKV. Consider using the optimizer hints to guide the optimizer to choose a better index or not to use index.',
+      index_join_build_side_too_large: 'This index join has a large build side. It might be slow and cause heavy pressure on TiKV. Consider using the optimizer hints to guide the optimizer to choose hash join.',
+      tikv_huge_table_scan: 'The TiKV read a lot of data. Consider using TiFlash to get better performance if it\'s necessary to read so much data.',
+    },
+  },
+}
+
+for (const key in translations) {
+  addTranslationResource(key, {
+    component: {
+      binaryPlan: translations[key],
+    },
+  })
+}
 
 export const DefaultNodeDetail = (nodeDetailProps) => {
   const nodeDatum = nodeDetailProps.data
+  const { t } = useTranslation()
 
   return (
-    <Collapse ghost defaultActiveKey={['1', '2']} destroyInactivePanel={true}>
-      <Collapse.Panel
-        header="Basic Info"
-        key="1"
-        className={styles.collapseHeader}
-      >
-        <div style={{ paddingLeft: 24 }} className={styles.BasicInfo}>
-          <p>
-            Actual Rows: <span>{nodeDatum.actRows}</span>
-          </p>
-          <p>
-            Estimate Rows: <span>{toFixed(nodeDatum.estRows, 0)}</span>
-          </p>
-          <p>
-            Run at: <span>{nodeDatum.storeType}</span>
-          </p>
-          <p>
-            Duration: <span>{nodeDatum.duration}</span>
-          </p>
-          {nodeDatum.cost && (
-            <p>
-              Cost: <span>{toFixed(nodeDatum.cost, 0)}</span>
-            </p>
-          )}
+    <Tabs defaultActiveKey="1" type="card" size="middle">
+      <Tabs.TabPane tab="General" key="1" style={{ padding: '1rem' }}>
```

Comment:
How about using a specific key like `general`, `hardware_usage` etc. So that will be easier to pick the tabs key in the future.